WcaOnRails/config/routes.rb

```diff
@@ -33,7 +33,6 @@
   get 'map' => 'competitions#embedable_map'
 
   get 'competitions/mine' => 'competitions#my_competitions', as: :my_comps
-  get 'competitions/for_senior(/:user_id)' => 'competitions#for_senior', as: :competitions_for_senior
```

Comment:
Hey @UnsolvedCypher 
Here is the problem of your tests failing. You removed this row.

Reply:
Thank you for catching that and sorry for my delay in response. I added this back in and tests are passing now. Please let me know if there are any other changes you would like.